scripts/technical-debt-metrics.sh

```diff
@@ -84,6 +84,8 @@ titlesPathsAndRegexes=(
   "disabled simpNF lints"          "*"      "nolint simpNF"
   "erw"                            "*"      "erw \["
   "maxHeartBeats modifications"    ":^MathlibTest" "^ *set_option .*maxHeartbeats"
+  "cases'"                         "*"      " cases' "
+  "induction'"                     "*"      " induction' "
```

Comment:
```suggestion
  "induction'"                     "*"      " induction' (?!\w+\s*:)"
```
I think is enough to catch only the `h :`-free versions? Though it's possible that the regex engine doesn't support `(?!`.

Reply:
On my computer, if I use `./scripts/technical-debt-metrics.sh`, then the `induction'` count gives 0.

If I try to type it out, using `!` gets interpreted as history lookup and if I try to escape it with `\!`, then the count returns 0.

Reply:
How about using `[^:]*$`?  It may miss some tech debt that should be counted, but it catches something for me: this is the relevant line
```bash
|1152|-140|induction'|
```